industrial_ci/src/workspace.sh

```diff
@@ -386,7 +386,7 @@ function ici_build_workspace {
     fi
 
     ici_step "setup_${name}_workspace" ici_prepare_sourcespace "$ws/src" "${sources[@]}"
-    ici_step "install_${name}_dependencies" ici_install_dependencies "$extend" "$ROSDEP_SKIP_KEYS" "$ws/src"
+    ici_step "install_${name}_dependencies" ici_install_dependencies "$extend" "$ROSDEP_SKIP_KEYS" "$ws/src" "${ROSDEP_SOURCE_FOLDERS[@]}"
```

Comment:
We might even move this into ici_install_dependencies, then it would always use the UNDERLAY by default

Reply:
I meant that it will always start from `UNDERLAY`, just like you intended.